db/migrate/20160331111827_add_stat_containerimage_create_rate_to_metric_rollups.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatContainerimageCreateRateToMetricRollups < ActiveRecord::Migration[5.0]
+  def change
+    add_column :metric_rollups, :stat_containerimage_new_images_rate, :integer
```

Comment:
The current columns use a four part naming that splits on `_`.  So, I'm thinking the new_images should be squished to newimages for consistency.  Or perhaps a better word than newimages (trying to think of one but I can't).

How is this stat different from create_rate, btw?